addons/website/static/src/js/utils.js

```diff
@@ -164,6 +164,7 @@ function prompt(options, _qweb) {
         field_name: '',
         'default': '', // dict notation for IE<9
         init: function () {},
+        cta_title: 'Continue',
```

Comment:
Maybe you can instead have a `buttons` field that lets the caller customize both button's text. Also don't forget to make the text translatable with `core._t(text)`, templates are automatically translatable but you have to be explicit about what is translatable in javascript